tests/unit/test_httpchecksum.py

```diff
@@ -106,20 +111,26 @@ def test_request_checksum_algorithm_model_opt_in(self):
             http_checksum={"requestAlgorithmMember": "Algorithm"}
         )
 
-        # Param is not present, no checksum will be set
+        # Param is not present, crc32 checksum will be set by default
```

Comment:
The test name isn't really correct anymore. This is specifically that we won't turn on checksums by default because it's breaking. Now that S3 has decided they're no longer upholding that, we're not checking that things are opt-in. This may be better as `test_request_checksum_algorithm_model_default`.

Reply:
Updated in 75db0fe6792083943d1af499275fbcfea77b3666. Thanks for calling this out.